packages/hmr/lib/injections/reload.ts

```diff
@@ -1,15 +1,13 @@
-import initClient from '../initializers/initClient';
+import initClient from '../initializers/initClient.js';
+import { HMR_ID } from '../consts.js';
 
-function addReload() {
+(() => {
   const reload = () => {
     chrome.runtime.reload();
   };
 
   initClient({
-    // @ts-expect-error That's because of the dynamic code loading
-    id: __HMR_ID,
```

Comment:
The `__HMR_ID` is an identifier injected by the `watch-rebuild-plugin` that is used to reload or refresh based solely on the context of where the build was run. You should not put this value as a constant.

```ts
    generateBundle(_options, bundle) {
      for (const module of Object.values(bundle)) {
        if (module.type === 'chunk') {
          module.code = `(function() {let __HMR_ID = "${id}";\n` + hmrCode + '\n' + '})();' + '\n' + module.code;
        }
      }
    },
  };
```

Reply:
Indeed, i've forgot about it.

Thanks for second look <3